<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioDebuggerWindow</class>
 <widget class="QMainWindow" name="AudioDebuggerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>578</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>160</width>
    <height>188</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>GBCEmulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="5" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>450</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_Square1">
        <property name="text">
         <string>Square 1:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_Square2">
        <property name="text">
         <string>Square 2:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_Noise">
        <property name="text">
         <string>Noise:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_Wave">
        <property name="text">
         <string>Wave:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="image_Square1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="image_Square2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="image_Wave">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="image_Noise">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen_ROM">
   <property name="text">
    <string>Open ROM</string>
   </property>
  </action>
  <action name="actionOpen_Debugger">
   <property name="text">
    <string>Open Debugger</string>
   </property>
  </action>
  <action name="actionVideo_Settings">
   <property name="text">
    <string>Video Settings</string>
   </property>
  </action>
  <action name="actionAudio_Settings">
   <property name="text">
    <string>Audio Settings</string>
   </property>
  </action>
  <action name="actionInput">
   <property name="text">
    <string>Input</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
